orchagent/vxlanorch.h

```diff
@@ -82,7 +82,7 @@ class VxlanTunnel
         return active_;
     }
 
-    bool createTunnel(MAP_T encap, MAP_T decap);
+    bool createTunnel(MAP_T encap, MAP_T decap, uint8_t encap_ttl=0);
```

Comment:
why do you set the default argument with value 0 instead of `VXLAN_ENCAP_TTL`?

Reply:
As Prince mentioned in another comment, VxLAN tunnel is created for different scenarios and for the VNET use-case we have default TTL set to 128 but for other use-cases all should work as before.